backend/src/zimfarm_backend/common/constants.py

```diff
@@ -116,6 +116,7 @@ def getenv(key: str, *, mandatory: bool = False, default: Any = None) -> Any:
 WASABI_WHITELIST_STATEMENT_ID = getenv(
     "WASABI_WHITELIST_STATEMENT_ID", default="ZimfarmWorkersIPsWhiteList"
 )
+WASABI_REQUEST_TIMEOUT = parse_timespan(getenv("WASABI_REQUEST_TIMEOUT", default="1m"))
```

Comment:
I suggest we set a more aggressive default value like 10s, this is already a lot and should be sufficient (or will be customized by env var if needed).